daisy_workflows/image_import/enterprise_linux/translate.py

```diff
@@ -239,11 +240,10 @@ def yum_install(g, *packages):
           '"{0}"'.format(p) for p in packages))
       return
     except Exception as e:
+      last_exception = e
       logging.debug('Failed to install {}. Details: {}.'.format(packages, e))
-  raise RuntimeError(
-      'Verify that you have specified the correct operating system '
-      'in the `--os` flag.  If you are bringing your own license (BYOL), '
-      'also verify that your subscription is eligible to be run on GCP.')
+  raise RuntimeError('Failed to install {}. Details: {}.'.format(
+      packages, last_exception))
 
```

Comment:
So is the previous error msg gone completely? It looks like the old one doesn't make sense at all

Reply:
"The previous msg" means "Verify that you have specified the correct operating system in the `--os` flag..."

Reply:
> So is the previous error msg gone completely?

The error message that you're referring to was added in https://github.com/GoogleCloudPlatform/compute-image-tools/pull/1066 . At that time, our analysis showed that the error message was appropriate. Since then, we added [1] which is a massive improvement, since it shows what's actually on the disk.

> It looks like the old one doesn't make sense at all

That's why we're removing it :)

1. https://github.com/GoogleCloudPlatform/compute-image-tools/blob/f5a91c65376c26d14c93854a42d408fc2da41a81/cli_tools/gce_vm_image_import/importer/errors.go#L28-L27